sources/input/scene_loader.c

```diff
@@ -5,35 +5,124 @@
 /*                                                    +:+ +:+         +:+     */
 /*   By: lfarias- <lfarias-@student.42.rio>         +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
-/*   Created: 2023/02/13 02:16:42 by lfarias-          #+#    #+#             */
-/*   Updated: 2023/03/07 19:28:57 by lfarias-         ###   ########.fr       */
+/*   Created: 2023/03/11 22:00:57 by lfarias-          #+#    #+#             */
+/*   Updated: 2023/03/13 16:26:19 by lfarias-         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "../../headers/mini_rt.h"
-#include <errno.h>
-#include <unistd.h>
-#include <fcntl.h>
+#include <stdio.h>
```

Comment:
Leftover header?

Reply:
yeah, my bad 2.0. was using the classic debug by printf and forgot about it.